profiles/krishnaUIDev.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/24648588/208228038-8be44f01-b8c7-4418-8a82-901d9e9628b3.png",
+  "issueId": 1567,
+  "name": "KrishnaKanth Kondoju",
+  "username": "krishnaUIDev"
+}  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Updated